providers/github/resources/github_repo.go

```diff
@@ -124,6 +124,90 @@ func (g *mqlGithubRepository) id() (string, error) {
 	return strconv.FormatInt(id, 10), nil
 }
 
+func initGithubMergeRequest(runtime *plugin.Runtime, args map[string]*llx.RawData) (map[string]*llx.RawData, plugin.Resource, error) {
+	var number int64
+	if x, ok := args["number"]; ok {
+		number, ok = x.Value.(int64)
+		if !ok {
+			return nil, nil, errors.New("wrong type for 'number' in github.mergeRequest initialization, it must be a number")
+		}
+	}
+	if number == 0 {
+		return nil, nil, errors.New("number must be set for github.mergeRequest initialization")
+	}
+	if len(args) > 2 {
```

Comment:
i think that should be 1 since you only need 1 argument to be provided in order to load a MR